src/libraries/System.Private.CoreLib/src/System/Reflection/StrongNameKeyPair.cs

```diff
@@ -29,9 +29,11 @@ public StrongNameKeyPair(byte[] keyPairArray)
         protected StrongNameKeyPair(SerializationInfo info, StreamingContext context) =>
             throw new PlatformNotSupportedException();
 
+        [Obsolete("Strong-name signing is not supported on this platform.")]
```

Comment:
Nit: Remove the hyphen so that this reads "Strong name signing". We don't traditionally hyphenate the term "strong name".
(same comment on line 36 below and in the ref assemblies)